assets/css/utilities.css

```diff
@@ -73,4 +73,20 @@
     100% {
     }
   }
+
+  .animate-fade-in-delayed {
```

Comment:
We already have `delay-100` above, doesn't that work? We can also define `delay-100` in the same way. Note that with the above we actually don't need to have `opacity-0` on the element, it is automatically hidden for the first 200ms and then shown.